net-libs/libotr/libotr-4.1.1.ebuild

```diff
@@ -17,6 +17,10 @@ RDEPEND="
 	dev-libs/libgpg-error:0="
 DEPEND="${RDEPEND}"
 
+PATCHES=(
+	"${FILESDIR}/${P}-fix-build-with-libgcrypt-1.10.patch"
```

Comment:
You didn't want the version number explicit in there? Like @arthurzam suggested:

```
"${FILESDIR}/${PN}-4.1.1-fix-build-with-libgcrypt-1.10.patch"
```

Reply:
Discussed on IRC, not really an issue...